templates/contrib/pc/rhel6.tpl.yaml

```diff
@@ -0,0 +1,157 @@
+{% set diskbus = diskbus |default("virtio") %}
```

Comment:
Do you plan to update this template with more changes? I would vote for reusing original rhel 6 jinja template and add `if` statement.

Reply:
Normally I would definitely follow your advice. 
In this case, I think, it would be better to explicitly make a copy and avoid introducing any changes in the original template to not hint that we support different machine types.
@fabiand wdyt?

Reply:
I would also generally follow Karel's advice.
In thit special case I'd also however go with a copy, because we really want to avoid that somebody starts to have the idea that the machine type can be anything else than q35. Therefore a copy would be appreciated.
We could actually consider to make it a real modified copy - no ansible templating, copy of the final template with a modified machine type.